install.sh

```diff
@@ -54,9 +54,6 @@
 #
 # __________ Process command-line arguments and environment variables _____________
 
-# Make sure we really exit with correct status when piping function output
-set -o pipefail
-
```

Comment:
That deletion was a mistake.  I'll cancel this pull request, commit a correction, and submit a new pull request.

I thought about merging master into the caffeinate-opencoarrays branch, but I didn't know how to do it in a way that preserves the version number on the branch so I cherry-picked this commit, assuming that would accomplish the same thing as merging.  Apparently it's not quite the same, given that the network graph doesn't depict the commit from master merging into the branch.